Source/Core/Core/IOS/USB/USBV5.cpp

```diff
@@ -119,11 +119,12 @@ std::optional<IPCReply> USBV5ResourceManager::GetDeviceChange(const IOCtlRequest
 
   std::lock_guard lk{m_devicechange_hook_address_mutex};
   m_devicechange_hook_request = std::make_unique<IOCtlRequest>(request.address);
-  // On the first call, the reply is sent immediately (instead of on device insertion/removal)
-  if (m_devicechange_first_call)
+  // If there are pending changes, the reply is sent immediately (instead of on device
+  // insertion/removal).
+  if (m_has_pending_changes)
   {
     TriggerDeviceChangeReply();
-    m_devicechange_first_call = false;
+    m_has_pending_changes = false;
```

Comment:
With the new "pending change" semantics, shouldn't this be reset at the end of TriggerDeviceChangeReply?

Reply:
Unless I'm missing something, this doesn't matter.

Reply:
The other TriggerDeviceChangeReply call seemed like it could be problematic, but I thought about this a bit more and went through a couple of scenarios and indeed this doesn't seem to matter.